backend/hct_mis_api/apps/administration/templates/admin/base.html

```diff
@@ -1,22 +1,41 @@
 {% extends "admin/base.html" %}
 
 {% block extrahead %}
-<script>
-    document.addEventListener("DOMContentLoaded", function (event) {
-        const url = window.location.href;
-        let color = '';
-        if (window.location.hostname === 'localhost') {
-            color = '#FF6600';
-        } else if (url.includes('trn')) {
-            color = '#BF360C';
-        } else if (url.includes('stg')) {
-            color = '#673AB7';
-        } else if (url.includes('dev')) {
-            color = '#00796B';
-        } else {
-            color = '#00ADEF';
-        }
-        document.getElementById("header").style.backgroundColor = color;
+  <script>
+    document.addEventListener('DOMContentLoaded', function(event) {
+      const url = window.location.href;
+      let color = '';
+      if (window.location.hostname === 'localhost') {
+        color = '#FF6600';
+      } else if (url.includes('trn')) {
+        color = '#BF360C';
+      } else if (url.includes('stg')) {
+        color = '#673AB7';
+      } else if (url.includes('dev')) {
+        color = '#00796B';
+      } else {
+        color = '#00ADEF';
+      }
+      document.getElementById('header').style.backgroundColor = color;
     });
-</script>
+  </script>
+  {% if MATOMO_TRACKER_URL and MATOMO_SITE_ID %}
+    <!-- Matomo -->
+    <script>
+      var _paq = window._paq = window._paq || [];
+      /* tracker methods like "setCustomDimension" should be called before "trackPageView" */
+      _paq.push(['trackPageView']);
+      _paq.push(['enableLinkTracking']);
+      (function() {
+        var u = '{{ MATOMO_TRACKER_URL }}';
+        _paq.push(['setTrackerUrl', u + 'matomo.php']);
```

Comment:
remove mathomo.php

Reply:
why matomo in the admin ?

Reply:
@saxix related to the task `include the following script in HOPE pages (FE/React and Admin/Django template)`, matomo should be implemented also in the admin. Should I remove it from the admin if it should be implemented only in the HOPE pages